swagger.yaml

```diff
@@ -59,6 +59,28 @@ paths:
         500:
           description: Internal Server Error
 
+  /nlp/search:
+    get:
+      tags:
+        - public
+      summary: Provided a query returns payload with search information about that query
+      description: Returns payload.
+      parameters:
+        - name: q
+          in: query
+          description: The query string
+          required: true
+          type: string
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Hub"
+        500:
+          $ref: '#/responses/InternalError'
+
```

Comment:
What's this for?

Reply:
Testing Functionality to check if the NLP apis work and what responses they return, but we discussed it's removal I've forgotten to remove it from the documentation
Will do